src/Services/Air/Air.js

```diff
@@ -13,9 +13,16 @@ module.exports = (settings) => {
   const log = (settings.options && settings.options.logFunction) || console.log;
   return {
     shop(options) {
+      if (options.async === true) {
+        return service.searchLowFaresAsync(options);
+      }
       return service.searchLowFares(options);
     },
 
+    retrieve(options) {
```

Comment:
Maybe it will make sense to be more specific with retrieve function, like `retrieveShop`? Otherwise, it may cause confusion with retrieve PNR

Reply:
Yes, I agree, there're too many retrieves like retrieveUR, retrieveTicket etc, 
I renamed the retrieve() with retrieveShop as you mentioned